integration/kubernetes/confidential/agent_image.bats

```diff
@@ -126,9 +132,15 @@ setup() {
 	echo $pod_config
 
 	create_test_pod
+	kubectl delete secret cococred --ignore-not-found
 }
 
 @test "$test_tag Test cannot pull an image from an authenticated registry with incorrect credentials" {
+	kubectl delete secret cococred --ignore-not-found
+	kubectl create secret docker-registry cococred --docker-server="https://quay.io/kata-containers/confidential-containers-auth" \
+	--docker-username="Arandomquaytestaccountthatdoesntexist" --docker-password="password"
+
+
 	if [ "${AA_KBC}" = "cc_kbc" ]; then
 		skip "As the test requires changing verdictd configuration and restarting its service"
 	fi
```

Comment:
Perhaps we could move line 139~141 after line 146 to skip create secret if AA_KBC=cc_kbc.

Reply:
yeah, good plan, I can do that and fixup later, but I don't want to interrupt the current test run now 🤞

Reply:
Done now :)